index.js

```diff
@@ -39,6 +33,17 @@ var lnCms = angular.module('lnCms', [
             name: route.state,
             url: route.url,
             templateUrl: 'templates/' + route.template + '/template.html',
+            controller: 'LnViewController as vm',
+            resolve: {
+              staticData:function(){
+                return lnCmsClientService.getStatic();
+              },
+              viewData: function(){
+                var endpoint = route.endpoint || 'post';
+                var params = route.params || {};
+                return lnCmsClientService.getData(endpoint, params);
```

Comment:
I think that there is something missing here.. The parameters that should be sent to the server are all these:

- route.params (fixed parameters, i.e. postId = 1)
- dynamic url parameters (i.e. `slug` for the url `/blog/:slug`)
- route.stateParams (state parameters which can change without changing the url, i.e. the filters in shootx search)